src/dynacli/bootstrap/init.py

```diff
@@ -0,0 +1,24 @@
+import os
+import shutil
+import stat
+
+
+def init(name: str, path: str) -> None:
+    """
+    Initialize the CLI entrypoint in given path
+
+    Args:
+        name (str): name of the new CLI tool
+        path (str): The path for CLI entrypoint
+
+    Return: None
+    """
+    wd = os.path.dirname(os.path.realpath(__file__))
+    cwd = path if path != "." else os.path.curdir
+    try:
+        shutil.copyfile(f"{wd}/_sample_cli", f"{cwd}/{name}")
```

Comment:
Not sure about this. My expectations would be that by typing dynacli init awesome you would create awesome folder with awesome script undneath, with path/root if defined and even a basic install script. What is done here is a bit too little. But considering time pressures, you may stage developing these features.